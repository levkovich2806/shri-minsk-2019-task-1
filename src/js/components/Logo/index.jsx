import React, { PureComponent } from "react";
import styles from "./index.module.scss";

class Logo extends PureComponent {
  render() {
    return (
      <a href="ya.ru" className={styles.logo}>
        <img src="images/logo.svg" alt="Логотоп" />
      </a>
    );
  }
}

export default Logo;
