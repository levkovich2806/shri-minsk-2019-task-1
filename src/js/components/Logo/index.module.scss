.logo {
  order: 1;
  flex: 1;
  margin: 18px;
}
