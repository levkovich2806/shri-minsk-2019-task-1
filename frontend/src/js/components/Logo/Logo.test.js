import React from "react";
import { mount } from "enzyme";
import Logo from "./index.jsx";
import renderer from "react-test-renderer";

describe("Logo", function () {
  test("Logo rendering", () => {

    const component = renderer.create(<Logo />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
