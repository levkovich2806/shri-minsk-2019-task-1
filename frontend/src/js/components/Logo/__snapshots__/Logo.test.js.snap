// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Logo rendering 1`] = `
<a
  className="logo"
  href="ya.ru"
>
  <img
    alt="Логотоп"
    src="images/logo.svg"
  />
</a>
`;
